src/Microsoft.Health.Fhir.Shared.Api/Features/ContentTypes/ContentTypeService.cs

```diff
@@ -139,5 +147,27 @@ private async Task<bool> IsFormatSupportedAsync(ResourceFormat resourceFormat)
                 }
             });
         }
+
+        private static string GetPrettyFromQueryString(HttpContext context)
```

Comment:
This looks pretty much like `GetFormatFromQueryString`. Is it possible to combine them to something like `GetValueFromQueryString(HttpContext context, string key)`?

Reply:
Good idea. Implemented in 86f3822.